recipes/torchani/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "torchani" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  git_url: git+https://github.com/aiqm/torchani.git#egg={{ version }}
```

Comment:
Why use the git URL rather than the tarball up on PyPI?

Reply:
I tried but the source tarball is messed up and not standard so install fails. I also tried the source dir from GH release but the installation failed too.

Reply:
Issue has been reported upstream https://github.com/aiqm/torchani/issues/507